camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -418,23 +423,23 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     this.userName = getUserName();
   }
 
-  timedifference (date: Date) {
+  timedifference(date: Date) {
```

Comment:
violates eslint rules `space-before-function-paren`. All function changes are violating this rule, which will be shown as a warning.

Reply:
After using prettier, again use eslint to fix this issue